### ISSUE TITLE
Suggestion: Catching prioritizing -Please make the walking more efficient when combined with 'pokemon to not catch'

### ISSUE THREAD

**Issue by berkie** [Ar1i/PokemonGo-Bot#462]
right now my trainer is running all over the place from caterpie to weedle and back while i have the bot set to not catch those.

Is it possible to make the bot only walk to pokemons it recognizes as 'to catch' ? maybe even allow certain priorities: let's say that 20 weedles and a dratini are in a 1km radius, i would naturally want the bot to prioritize the dratini over the weedles. if there were multiple dratini's, i would want it to prioritize the one with the highest IV. If those dratini's have similar IV's, let's say they differ less than 10%, i would want to prioritize CP.)

TL;DR: please make the bot determine if it's worth walking to a pokemon before walking (is it on the to catch list or not), a list of pokemon to prioritize in that order, and then by either IV and/or CP.
